server.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: "WordPress Server: Install LEMP Stack with PHP 5.6 and MariaDB MySQL"
+  hosts: web
+  remote_user: root
```

Comment:
I anticipated you'd change this to `sudo: yes` and `remote_user: admin`, given that `secure-root.yml` blocks ssh access for root.